tidb-cloud/use-chat2query-api.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Get Started with Chat2Query API
+summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL statements using AI by providing instructions.
+---
+
+# Get Started with Chat2Query API
+
+TiDB Cloud provides the Chat2Query API, a RESTful interface that allows you to generate and execute SQL statements using AI by providing instructions. Then, the API returns the query results for you.
+
+Chat2Query API can only be accessed through HTTPS, ensuring that all data transmitted over the network is encrypted using TLS.
+
+> **Note:**
+>
+> Chat2Query API is only available for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
+
+## Before you begin
+
+Before using the Chat2Query API, make sure that you have created a [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) cluster. If you do not have one, follow the steps in [Create a cluster](/tidb-cloud/create-tidb-cluster.md) to create one.
+
+## 1. Enable the Chat2Query API
+
+To enable the Chat2Query API, perform the following steps:
+
+1. Go to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
+
+    > **Tip:**
+    >
+    > If you have multiple projects, you can view the project list and switch to another project from the ☰ hover menu in the upper-left corner.
+
+2. Click your cluster name, and then click **Chat2Query** in the left navigation pane.
+3. In the upper-right corner of Chat2Query, click **...** and select **Settings**.
+4. Enable **DataAPI** and the **you can view and manage Chat2Query API in Data Service** message is displayed.
```

Comment:
```suggestion
4. Enable DataAPI, and then click **Save**.
```

Reply:
The **you can view and manage Chat2Query API in Data Service** message might be changed at any time in the future. Avoid stating that clearly in the doc to prevent the doc to be outdated easily.